app/(api)/api/collections/[collection_type]/[id]/get.ts

```diff
@@ -0,0 +1,9 @@
+import { NextRequest } from 'next/server';
+import { findCollectionItem } from '../../../../_datalib/collections/findCollectionItem';
```

Comment:
I think you can say @datalib. Check the tsconfig.json to see what shortcuts you can use.